app/src/main/kotlin/no/nav/tiltakspenger/saksbehandling/meldekort/domene/Meldeperiode.kt

```diff
@@ -29,7 +26,10 @@ data class Meldeperiode(
     val antallDagerForPeriode: Int,
     val girRett: Map<LocalDate, Boolean>,
     val sendtTilMeldekortApi: LocalDateTime?,
-) {
+) : Comparable<Meldeperiode> {
+    val ingenDagerGirRett = girRett.values.none { it }
+    val antallDagerSomGirRett = girRett.values.count { it }
```

Comment:
Liten optimalisering!
```suggestion
    val antallDagerSomGirRett = girRett.values.count { it }
    val ingenDagerGirRett = antallDagerSomGirRett == 0
```